core/templates/pages/exploration-editor-page/services/exploration-warnings.service.ts

```diff
@@ -235,6 +238,39 @@ export class ExplorationWarningsService {
     return indexes;
   }
 
+  _getStatesWithInvalidRedirection(
+      initStateId: string, links: GraphLink[]
+  ): string[] {
+    let results = [];
+    let states = this.explorationStatesService.getStates();
+    let bfsStateList = this.computeGraphService.computeBfsTraversalOfStates(
+      initStateId, states, initStateId);
+    let seen: Record<string, boolean> = {};
```

Comment:
As a reader, it is unclear what "seen" represents here. Can we rename it?

Reply:
Done.